src/ProductConstructionService/ProductConstructionService.DependencyFlow/PullRequestUpdater.cs

```diff
@@ -933,6 +951,32 @@ private static TimeSpan GetReminderDelay(DateTime updatedAt)
         };
     }
 
+    private async Task registerSubscriptionUpdateAction(SubscriptionUpdateAction subscriptionUpdateAction,
```

Comment:
When we break arguments to separate lines, we also break the first one (in VS there's the indentation refactoring)